### PR TITLE
All lights turn red in emergency procedures

### PR DESCRIPTION
**Pull request by warface1234455**
# Document the changes in your pull request
Delta, Gamma and *@&!#&EPSILON*#%@ alert now make all lights turn red

# Why is this good for the game?
For immersion and make the station seems like its on actual emergency procedure

# Testing
Tested,
switched to delta/epsilon alert level, all lights turned red
switched back to normal alert level, all lights went back to normal

Tried to use fire alarm to turn off red lights on delta, didnt do that and instead it triggered firelocks, then hit the fire alarm again It opened all the firelocks and the lights still stayed red as intended

image

Shouldnt make shuttle lights go red

# Changelog

:cl:  
tweak: Delta, Gamma and *@&!#&EPSILON*#%@ alert now make all lights turn red
/:cl:

### CODE REVIEW

**Comment by solwashere** [comment]
It would be best to test this to ensure that it doesn't break the lighting

**Comment by ShoziOffical** [comment]
> It would be best to test this to ensure that it doesn't break the lighting

no it doesnt break it

**Comment by solwashere** [comment]
> > It would be best to test this to ensure that it doesn't break the lighting
> 
> no it doesnt break it

It would be nice to know that before it's merged

**Comment by ShoziOffical** [comment]
> > > It would be best to test this to ensure that it doesn't break the lighting
> > 
> > 
> > no it doesnt break it
> 
> It would be nice to know that before it's merged

mhm

**Comment by MajManatee** [comment]
now we need dramatic music to play

**Comment by solwashere** [comment]
> now we need dramatic music to play

https://youtu.be/XoUfsbU-wfA?si=OehmATHMuhLYgjmg

**Comment by ShoziOffical** [comment]
> now we need dramatic music to play

would do that if custom timer didnt exist-

**Comment by MajManatee** [comment]
I was thinkin the SCP: SL music when the nuke goes off

**Comment by ShoziOffical** [comment]
> I was thinkin the SCP: SL music when the nuke goes off

well yeahhhhh if the timer sync with it

**Comment by N3D6** [comment]
> > It would be best to test this to ensure that it doesn't break the lighting
> 
> no it doesnt break it

local hosting isn’t conclusive enough for these types of things

**Comment by ShoziOffical** [comment]
> > > It would be best to test this to ensure that it doesn't break the lighting
> > 
> > 
> > no it doesnt break it
> 
> local hosting isn’t conclusive enough for these types of things

how is it breaking lights if im using the same fire alarm proc

**Comment by ToasterBiome** [comment]
> > > It would be best to test this to ensure that it doesn't break the lighting
> > 
> > 
> > no it doesnt break it
> 
> local hosting isn’t conclusive enough for these types of things

false

**Comment by N3D6** [comment]
> > > > It would be best to test this to ensure that it doesn't break the lighting
> > > 
> > > 
> > > no it doesnt break it
> > 
> > 
> > local hosting isn’t conclusive enough for these types of things
> 
> false

my source was: trust me bro
i feel like every light changing at once would probably not be light on the server

**Comment by azzzertyy** [comment]
> > > > > It would be best to test this to ensure that it doesn't break the lighting
> > > > 
> > > > 
> > > > no it doesnt break it
> > > 
> > > 
> > > local hosting isn’t conclusive enough for these types of things
> > 
> > 
> > false
> 
> my source was: trust me bro i feel like every light changing at once would probably not be light on the server

Circardian Rhythm
We literally already do this ned

**Comment by Maia-J** [comment]
> > > > > > It would be best to test this to ensure that it doesn't break the lighting
> > > > > 
> > > > > 
> > > > > no it doesnt break it
> > > > 
> > > > 
> > > > local hosting isn’t conclusive enough for these types of things
> > > 
> > > 
> > > false
> > 
> > 
> > my source was: trust me bro i feel like every light changing at once would probably not be light on the server
> 
> Circardian Rhythm We literally already do this ned

And power shutdowns

**Comment by ShoziOffical** [comment]
> > > > > > > It would be best to test this to ensure that it doesn't break the lighting
> > > > > > 
> > > > > > 
> > > > > > no it doesnt break it
> > > > > 
> > > > > 
> > > > > local hosting isn’t conclusive enough for these types of things
> > > > 
> > > > 
> > > > false
> > > 
> > > 
> > > my source was: trust me bro i feel like every light changing at once would probably not be light on the server
> > 
> > 
> > Circardian Rhythm We literally already do this ned
> 
> And power shutdowns

yeah blackout does this just fine

**Comment by Addust** [comment]
are ruins affected?

**Comment by ShoziOffical** [comment]
> are ruins affected?

No, station z level only but yeah asteroid map has multiple ruins in its z so ill make a check for it as well

**Review by Moltijoe — CHANGES_REQUESTED** [review]
please remove the "shuttle call during red alert" part

**Comment by warface1234455** [comment]
> please remove the "shuttle call during red alert" part

till i get a reason why

File: code/modules/shuttle/emergency.dm [hunk 1]
```diff
@@ -266,6 +266,7 @@
 		SSshuttle.emergencyLastCallLoc = signalOrigin
 	else
 		SSshuttle.emergencyLastCallLoc = null
+
```
Comment (ShoziOffical) [inline code/modules/shuttle/emergency.dm:269]: welp

**Review by Moltijoe — APPROVED** [review]

**Pull request merged**